### DISCUSSION TITLE
Release Announcment

### DISCUSSION THREAD

**Discussion by misterek** [bottlerocket-os/bottlerocket#4096] [General]
Is there any chance that we can announce the release, via github or similar, slightly before the AWS AMI gets updated?

Our nodes started to get replaced at about 4:40 Mountain time (40 minutes ago) with Bottlerocket 1.20.4.  It looks like the release was just announced 1 minute ago.   This is very handy/necessary for understanding what is in the nodes that are getting updated.

Thanks!

**Comment by ginglis13** [comment]
Hi @misterek,

We always update our [changelog](https://github.com/bottlerocket-os/bottlerocket/blob/develop/CHANGELOG.md) with details of what is landing in an upcoming release. The changelog section pertaining to a given release is the same information as included on the releases page: https://github.com/bottlerocket-os/bottlerocket/releases. 

We strive to keep the window between AMI release and release announcement on GitHub as small as possible, but as you point out, that window is not always consistent. The reverse situation here is not ideal either - an hour delay between saying we've released something, without any AMI published for that release, is not a great experience. Going forward, we can attempt to keep release tracking issues such as https://github.com/bottlerocket-os/bottlerocket/issues/4069 (unfortunately looks like we missed a tracking issue for 1.20.4) up to date with comments that provide status updates for a release. That way, you could subscribe to that issue for notifications.

Would that work for you?
